app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/dashboard/view/MyDashboardFragment.kt

```diff
@@ -109,7 +110,11 @@ class MyDashboardFragment : Fragment() {
     fun showMessage() {
         dashboardViewModel.showMsg.observe(this, androidx.lifecycle.Observer {
             val message = it?.showMsg
-            Toast.makeText(context, message, Toast.LENGTH_LONG).show()
+            view?.let { it1 ->
+                if (message != null) {
+                    Snackbar.make(it1, message, Snackbar.LENGTH_LONG).show()
+                }
+            }
```

Comment:
Use existing utils method
'view?.showSnackbar(message!!)'
 instead of 
Snackbar.make(it1, message, Snackbar.LENGTH_LONG).show()

Add an extra parameter in ViewUtils if you need.